<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="anonymiser.qrc">
    <normaloff>:/pixmaps/small_masks.png</normaloff>:/pixmaps/small_masks.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="SaveOpenText">
       <property name="text">
        <string>Opening...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="folderNameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>FolderName</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savePushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>85</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setStringsButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>87</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Set strings</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>261</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="NameDoBInputLine">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="newNameLabel">
         <property name="text">
          <string>New Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="newNameBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="NewDobLabel">
         <property name="text">
          <string>New DoB:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="newDobBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>285</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="radioFolder">
       <property name="text">
        <string>Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioZip">
       <property name="text">
        <string>Zip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="anonPushButton">
       <property name="text">
        <string>Anonymise and Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>109</width>
      <height>136</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="openFolderPushButton">
       <property name="toolTip">
        <string>Open a folder with your DICOMs</string>
       </property>
       <property name="text">
        <string>Open Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeFilePushbutton">
       <property name="toolTip">
        <string>Close your DICOM file</string>
       </property>
       <property name="text">
        <string>Close File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newPushButton">
       <property name="toolTip">
        <string>Close your DICOM file and clear anonymisation names</string>
       </property>
       <property name="text">
        <string>Close and Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="toolTip">
        <string>About this tool</string>
       </property>
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>savePushButton</zorder>
   <zorder>setStringsButton</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>folderNameLabel</zorder>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="anonymiser.qrc"/>
 </resources>
 <connections/>
</ui>
